lib/OpenQA/Worker/Job.pm

```diff
@@ -196,8 +196,6 @@ sub _compute_timeouts ($job_settings) {
     my $timeout_scale = $job_settings->{TIMEOUT_SCALE};
     $max_job_time = DEFAULT_MAX_JOB_TIME unless looks_like_number $max_job_time;
     $max_setup_time = DEFAULT_MAX_SETUP_TIME unless looks_like_number $max_setup_time;
-    # disable video for long-running scenarios by default
-    $job_settings->{NOVIDEO} = 1 if !exists $job_settings->{NOVIDEO} && $max_job_time > DEFAULT_MAX_JOB_TIME;
```

Comment:
I think something like this logic is still beneficial. How about

```suggestion
    # disable video for long-running scenarios by default
    $job_settings->{NOVIDEO} = 1 if !exists $job_settings->{NOVIDEO} && $max_job_time > DEFAULT_MAX_VIDEO_JOB_TIME;
```

with DEFAULT_MAX_VIDEO_JOB_TIME being something like 4*DEFAULT_MAX_JOB_TIME so that there is still some reasonable limit for overly excessive values of MAX_JOB_TIME?